hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/mapred/TestTableInputFormat.java

```diff
@@ -147,7 +147,7 @@ public static Table createTable(byte[] tableName, byte[][] families) throws IOEx
    * @param key
    * @param expectedKey
    * @param expectedValue
-   * @return
+   * @return true if succeed
```

Comment:
What succeeds? Would be great if it was in the format `true if xxx succeeds, false otherwise`.